projects/storefrontapp/src/app/app.module.ts

```diff
@@ -28,16 +28,24 @@ if (!environment.production) {
   devImports.push(StoreDevtoolsModule.instrument());
 }
 
+let additionalImports = [];
+
+if (environment.cds) {
+  additionalImports = [...additionalImports, ...cdsFeature.imports];
+}
+
+if (environment.b2b) {
```

Comment:
We should merge this branch into `epic/b2b`. Some files to consider:
- https://github.com/SAP/spartacus/pull/7741/files#diff-827d378f5be063c804b5d194b664a979
- https://github.com/SAP/spartacus/pull/7741/files#diff-67849fba1a80b4db7314a20c4f560c38

Reply:
I will leave that to them to sync with develop after we merge it.